src/frontend/contexts/ProjectContext.tsx

```diff
@@ -5,15 +5,23 @@ import {usePersistedProjectId} from '../hooks/persistedState/usePersistedProject
 
 type MapeoProject = Awaited<ReturnType<MapeoClientApi['getProject']>>;
 
-type ActiveProjectContext = {
-  project: MapeoProject | undefined;
+type ActiveProjectContextType = {
+  project?: MapeoProject;
```

Comment:
Not sure it'll make a difference in our case, but there is a notable distinction between the before and after:

https://www.typescriptlang.org/play?#code/C4TwDgpgBAYg9nKBeKBvAUFKYBOcBWEAxsAPwBcUAzsDgJYB2A5gNzoC+66okUAQgEMcyNJmx5CJSjXrMoAHygBXBgBMIAM0YRVbTuiJwGNKMErxEKVPsPHgpgEyVBwq+yA

basically the original definition ensures that the property exists on the value that's annotated with the type (even it it's undefined), which is helpful in certain cases. would lean towards keeping the original definition if it's not affecting anything

```suggestion
  project: MapeoProject | undefined;
```

Reply:
https://github.com/digidem/CoMapeo-mobile/pull/89/commits/1a9a0ef294e7d4226dc3159d35cf89c0a973022e